### PR TITLE
add delete rowaction

### PR DESCRIPTION
**Pull request by RobDolares**
Signed-off-by: rhowingt@redhat.com <rhowingt@redhat.com>

### CODE REVIEW

File: frontend/src/lib/delete-policy.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        },
+    ]
+    // resources.push({
+    //     apiVersion: PlacementBindingApiVersion,
+    //     kind: PlacementBindingKind,
+    //     metadata: { name: policy.metadata.name },
+    // })
+    const deleteResourcesResult = deleteResources(resources)
+
+    return {
```
Comment (jamestalton) [inline frontend/src/lib/delete-policy.ts:25]: Think there is a helper function for this...

File: frontend/src/resources/policy.ts [hunk 2]
```diff
@@ -8,6 +8,8 @@ export type PolicyApiVersionType = 'policy.open-cluster-management.io/v1'
 export const PolicyKind = 'Policy'
 export type PolicyKindType = 'Policy'
 
+export const PolicyPluralKind = 'Policies'
```
Comment (jamestalton) [inline frontend/src/resources/policy.ts:11]: delete these

File: frontend/src/resources/resource.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export interface IResource extends IResourceDefinition {
     apiVersion: string
     kind: string
     metadata?: Metadata
+    plural?: string
```
Comment (jamestalton) [inline frontend/src/resources/resource.ts:16]: remove this

File: frontend/src/lib/delete-policy.ts (hunk 1 above)
Comment (RobDolares) [reply frontend/src/lib/delete-policy.ts:23]: not able to locate helper function for the promise, can wait to add that in this PR or next iteration.

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**